terraform/email/main.tf

```diff
@@ -49,7 +59,16 @@ resource "aws_route53_record" "primary_amazonses_dmarc_record" {
   name    = "_dmarc.${var.domain}"
   type    = "TXT"
   ttl     = "60"
-  records = ["v=DMARC1; p=none; rua=${var.dmarc}; fo=1; adkim=s; aspf=s"]
+  records = ["v=DMARC1; p=none; rua=${var.dmarc}; fo=1; adkim=s; aspf=r"]
```

Comment:
I think we want to set ``adkim=r`` here as well. I believe that our TestPyPI emails need relaxed alignment, and that explains why we're not at 100% alignment for DKIM.

Reply:
done.